contracts/hardhat.config.ts

```diff
@@ -31,7 +32,7 @@ const config: HardhatUserConfig = {
     } as any,
     rinkeby: {
       url: process.env.RINKEBY_JSONRPC_HTTP_URL || 'http://127.0.0.1:8545',
-      accounts: { mnemonic: WALLET_MNEMONIC },
+      accounts: [WALLET_PRIVATE_KEY],
```

Comment:
I think this is causing a hardhat build error for the build. 
<img width="923" alt="Screen Shot 2021-08-29 at 2 43 36 PM" src="https://user-images.githubusercontent.com/15589226/131264784-44cfc1a7-db65-4573-bfa6-084219eec35f.png">